pkg/controller/group/group_controller_test.go

```diff
@@ -0,0 +1,80 @@
+package group
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	userv1 "github.com/openshift/api/user/v1"
+	"github.com/openshift/osd-metrics-exporter/pkg/metrics"
+	"github.com/prometheus/client_golang/prometheus/testutil"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+func TestReconcileGroup_Reconcile(t *testing.T) {
+	for _, tc := range []struct {
+		name        string
+		expectError bool
```

Comment:
`true` not tested (but should be?)

Reply:
Difficult to simulate tests for this so dropping it for now.